java/src/jmri/jmrit/operations/routes/RouteEditTableModel.java

```diff
@@ -440,31 +436,17 @@ public int getLastMaxTrainLength() {
         return _maxTrainLength;
     }
 
-    private void setWait(Object value, RouteLocation rl) {
-        int wait;
-        try {
-            wait = Integer.parseInt(value.toString());
-        } catch (NumberFormatException e) {
-            log.error("Location wait must be a number");
-            JmriJOptionPane.showMessageDialog(null, Bundle.getMessage("EnterWaitTimeMinutes"), Bundle
-                    .getMessage("WaitTimeNotValid"), JmriJOptionPane.ERROR_MESSAGE);
-            return;
-        }
-        rl.setWait(wait);
+    private void setTravel(Object value, RouteLocation rl) {
+        int wait = (int) value;
+        rl.setWait(wait - Setup.getTravelTime());
```

Comment:
Can wait be less than Setup.getTravelTime() ?
Maybe add a check that the value is >= 0?

Reply:
Can wait be less than Setup.getTravelTime() ?

Yes. For example say the user set the average travel time to 10 minutes, and value -5, that makes the travel time 5 minutes between stations.